### PR TITLE
MAINT: Changes to TD window

### PR DESCRIPTION
**Pull request by adler-j**
* Change code to follow original article
* Fix bug with default `helical_geometry`
* Rename `n_half_rot` to the more standard `n_pi`

### CODE REVIEW

File: odl/tomo/analytic/filtered_back_projection.py [hunk 1]
```diff
@@ -163,19 +169,18 @@ def tam_danielson_window(ray_trafo, smoothing_width=0.05, n_half_rot=1):
...
-    theta = phi * 2
+    dx_axis = dx * src_radius / (src_radius + det_radius)
 
-    # Compute lower and upper bound
-    source_to_line_distance = src_radius + src_radius * np.cos(theta)
-    scale = (src_radius + det_radius) / source_to_line_distance
+    def Vn(u):
+        return (pitch / (2 * np.pi) *
+                (1 + (u / src_radius) ** 2) *
+                (n_pi * np.pi / 2.0 - np.arctan(u / src_radius)))
```
Comment (kohr-h) [inline odl/tomo/analytic/filtered_back_projection.py]: Not sure I wanna go read the paper to see what this function does. Is there some intuition behind it? A comment or a docstring would help here.

Comment (adler-j) [reply odl/tomo/analytic/filtered_back_projection.py]: Frankly, I dunno. The old code was me with pen and paper (I didn't even know the method existed and had a name at that time), trying to draw in 3D.

Interestingly, it seems that this change does not do anything to the output of the function, but merely harmonizes it with the article.

Comment (adler-j) [reply odl/tomo/analytic/filtered_back_projection.py]: I can add the general idea that this is the helix that the source travels on, projected onto the detector.

Comment (kohr-h) [reply odl/tomo/analytic/filtered_back_projection.py]: > I can add the general idea that this is the helix that the source travels on, projected onto the detector.

That'd be a great help!

**Comment by kohr-h** [comment]
> isn't this more of a BUG rather than a MAINT?

Oh I see now that one of the commits has BUG. Never mind.

**Comment by kohr-h** [comment]
> SyntaxError: Non-ASCII character '\xc3' in file ...

Why don't we add `# coding: utf-8` to all files we modify? It won't last for very long.

File: odl/tomo/analytic/filtered_back_projection.py [hunk 2]
```diff
@@ -138,6 +139,11 @@ def tam_danielson_window(ray_trafo, smoothing_width=0.05, n_half_rot=1):
     *Exact cone beam CT with a spiral scan*.
     Physics in Medicine & Biology 4 (1998), p 1015.
     https://dx.doi.org/10.1088/0031-9155/43/4/028
+
+    [PKGT2000] Proksa R, Kohler T, Grass M, Timmer J.
```
Comment (kohr-h) [inline odl/tomo/analytic/filtered_back_projection.py]: Disagree with replacing "Köhler" with "Kohler", it's wrong. Please use the unicode marker instead.

Comment (adler-j) [reply odl/tomo/analytic/filtered_back_projection.py]: You caught me :( I'll fix

Comment (kohr-h) [reply odl/tomo/analytic/filtered_back_projection.py]: Don't mess with German names :wink:

**Pull request merged**